fluent-package/templates/package-scripts/fluent-package/deb/postinst

```diff
@@ -137,12 +125,6 @@ migration_from_v4_post_process() {
         rm -fr /var/log/<%= compat_package_dir %>
         ln -sf /var/log/<%= package_dir %> /var/log/<%= compat_package_dir %>
     fi
-    if [ "$v4migration_with_restart" = "y" ]; then
-        if [ -n "$(command -v systemctl)" ]; then
-            systemctl daemon-reload
-            systemctl restart <%= service_name %>
-        fi
-    fi
 }
```

Comment:
v4migration section should not be touched because it changes existing behavior.

Reply:
If it is not appropriate might be better to fix in another PR.

Reply:
Thanks. I dropped the commit from this PR.